src/include/cpu/rpp_cpu_common.hpp

```diff
@@ -3028,6 +3028,22 @@ inline void compute_color_cast_12_host(__m128 *p, __m128 pMul, __m128 *pAdd)
     p[2] = _mm_fmadd_ps(_mm_sub_ps(p[2], pAdd[2]), pMul, pAdd[2]);    // color_cast adjustment Rs
 }
 
+inline void compute_color_temperature_48_host(__m256 *p, __m256 pAdj)
```

Comment:
please rename the variable name to snake_case

Reply:
Rajy will get back on this.

Reply:
Performance numbers are similar to using templated function for each datatype with minor differences due to noise. Modified them accordingly.

Reply:
Rajy, this is actually consistent with all other API. Camel case for variables and snake case for RPP API.